### ISSUE TITLE
Handle 'withdrawn' items better

### ISSUE THREAD

**Issue by tseemann** [enasequence/enaBrowserTools#16]
I wrote this tutorial last year:
https://github.com/ngs-docs/angus/blob/2017/kraken_species_identification.rst

So i tried to use your tool to download it.
```
enaDataGet -f fastq SRR2423672
ERROR: Something unexpected went wrong please try again.
If problem persists, please contact datasubs@ebi.ac.uk for assistance.
```
Now it turns out the entry used to exist, but was withdrawn:

*Run: SRR2423672 has been removed at the submitter's request.*
See: http://www.ebi.ac.uk/ena/data/view/SRR2423672

Are you able to catch this situation better and inform the user?

**Comment by nicsilvester** [comment]
It should be saying record not available.  I'll check why this didn't happen. Thanks for the heads up

**Comment by nicsilvester** [comment]
Fixed in release v1.3.1
